test-libfuzzer-target.c

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2019 Google Inc. All rights reserved.
```

Comment:
nit: not an expert myself, but I'd guess it should be "Google LLC" since it's being added with 2019 year. Older files were left with "Google Inc" since they were created back then